content/02.main-text.md

```diff
@@ -63,5 +63,5 @@ The objectives of the action plan for each sector are as follows:
 |**ADAPTATION OBJECTIVES: Protect accelerated growth pillars**  |**ADAPTATION MEASURES AT SHORT, MEDIUM AND LONG-TERM**|
 |--|--|
 | Agriculture | -	Cultivate early varieties or drought-resistant crops (Short) - Apply water and soil conservation methods (stone barriers, small dikes, filtering dikes, terraces, half moons, agroforestry, dune fixing etc.) (Short) - Promote sustainable land management (SLM) (Medium) - Improve access to climate information  (Medium) - Introduce agricultural insurance (Long) |
-|Livestock farming | -	Fight bush fires in order to prevent destruction of dry-season grazing reserves-  Adopt best animal husbandry and pastoral practices (pastoral hydraulics, pastoral resource management, pasture mowing and conservation, pasture crops, silage, animal mobility and transhumance etc.) - Ensure stakeholders take account of climate variability in development project and programme planning by improving their skills - Preserve cattle breeding at serious risk from climate variability  - Ensure farmers adopt animal production methods adapted to a hot climate|
+|Livestock farming | -	Fight bush fires in order to prevent destruction[@doi-org.ezproxy3.lib.le.ac.uk/10.1136/bmj.312.7031.619] of dry-season grazing reserves-  Adopt best animal husbandry and pastoral practices (pastoral hydraulics, pastoral resource management, pasture mowing and conservation, pasture crops, silage, animal mobility and transhumance etc.) - Ensure stakeholders take account of climate variability in development project and programme planning by improving their skills - Preserve cattle breeding at serious risk from climate variability  - Ensure farmers adopt animal production methods adapted to a hot climate|
```

Comment:
```suggestion
|Livestock farming | -	Fight bush fires in order to prevent destruction [@doi:10.1136/bmj.312.7031.619] of dry-season grazing reserves-  Adopt best animal husbandry and pastoral practices (pastoral hydraulics, pastoral resource management, pasture mowing and conservation, pasture crops, silage, animal mobility and transhumance etc.) - Ensure stakeholders take account of climate variability in development project and programme planning by improving their skills - Preserve cattle breeding at serious risk from climate variability  - Ensure farmers adopt animal production methods adapted to a hot climate|
```

I reformatted the DOI citation.  Typically, the type of identifier (`doi`) is added to the front of the identifier and `:` separated.

There are more examples of citation-by-identifier at https://github.com/manubot/rootstock/blob/main/USAGE.md#citations